### PR TITLE
AP_GPS:add note to old param names to avoid confusion

### PR DESCRIPTION
**Pull request by Hwurzburg**
Addresses #27929

### CODE REVIEW

**Review by peterbarker — APPROVED** [review]
LGTM

File: libraries/AP_GPS/AP_GPS.cpp [hunk 1]
```diff
@@ -1999,111 +1999,111 @@ bool AP_GPS::gps_yaw_deg(uint8_t instance, float &yaw_deg, float &accuracy_deg,
...
-    // @Description: Bitmask for what GNSS system to use on the second GPS (all unchecked or zero to leave GPS as configured)
+    // @DisplayName: GNSS system configuration.
+    // @Description: Bitmask for what GNSS system to use on the second GPS (all unchecked or zero to leave GPS as configured). Renamed in 4.6 and later to GPS2_GNSS_MODE
     // @Bitmask: 0:GPS,1:SBAS,2:Galileo,3:Beidou,4:IMES,5:QZSS,6:GLONASS
     // @User: Advanced
 
     // @Param: _RATE_MS
-    // @DisplayName: GPS update rate in milliseconds
-    // @Description: Controls how often the GPS should provide a position update. Lowering below 5Hz(default) is not allowed. Raising the rate above 5Hz usually provides little benefit and for some GPS (eg Ublox M9N) can severely impact performance.
+    // @DisplayName: GPS update rate in milliseconds. Renamed in 4.6 to GPS1_RAATE_MS
```
Comment (nexton-winjeel) [inline libraries/AP_GPS/AP_GPS.cpp:2025]:
```suggestion
    // @DisplayName: GPS update rate in milliseconds. Renamed in 4.6 to GPS1_RATE_MS
```

Comment (Hwurzburg) [reply libraries/AP_GPS/AP_GPS.cpp:2025]: you are right...sorry this was missed...will PR another change to fix

Comment (Hwurzburg) [reply libraries/AP_GPS/AP_GPS.cpp:2025]: seems its been fixed already

**Pull request merged**